portafly/src/components/data-list/DataListContext.tsx

```diff
@@ -35,29 +34,25 @@ export interface IDataListContext {
   dispatch: React.Dispatch<Action<any>>
 }
 
-const DataListContext = createContext<IDataListContext>({ state: initialState, dispatch: () => {} })
+const DataListContext = createContext<IDataListContext>({ state: defaultState, dispatch: () => {} })
 
 type Props = {
-  data: {
-    columns: DataListCol[]
-    rows: DataListRow[]
+  initialState: {
+    table: {
+      columns: DataListCol[]
+      rows: DataListRow[]
+    }
```

Comment:
Maybe directly:
```suggestion
type Props = {
    table: {
      columns: DataListCol[]
      rows: DataListRow[]
    },
    pagination: ... // future
    filters: ... // future
```

Or are we expecting something else than an initial state?